### PR TITLE
RA-1842: Follow Update of module versions in Ref-app-distro

### PR DESCRIPTION
**Pull request by sherrif10**
https://issues.openmrs.org/browse/RA-1842
 cc @dkayiwa  @mozzy11

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -36,17 +36,17 @@
 		<uiframeworkVersion>3.19.0</uiframeworkVersion>
 		<registrationcoreVersion>1.10.0</registrationcoreVersion>
 		<registrationappVersion>1.20.0</registrationappVersion>
-		<eventVersion>2.9.0-SNAPSHOT</eventVersion>
-		<idgenVersion>4.6.0</idgenVersion>
+		<eventVersion>2.8.0</eventVersion>
+		<idgenVersion>4.6.0-SNAPSHOT</idgenVersion>
```
Comment (dkayiwa) [inline pom.xml:40]: Why did you change the above to snapshot?

Comment (sherrif10) [reply pom.xml:40]: For event module, latest changes are in the current version, No significant changes have been made since last release. 
For Idgen module: it has not been changed  in this PR , Am still looking into it

Comment (dkayiwa) [reply pom.xml:40]: You do not seem to have understood my question. Can you look at it again together with the changes in this pull request?

Comment (mozzy11) [reply pom.xml:40]: haha @sherrif10 can you look again more closer ??

Comment (mozzy11) [reply pom.xml:40]: really  >><idgenVersion>4.6.0-SNAPSHOT</idgenVersion>   ??

Comment (sherrif10) [reply pom.xml]: oops sorry , i resolved that
